css/styles.css

```diff
@@ -0,0 +1,445 @@
+  font-weight: 300;
+  margin-right: 1rem;
+}
+
+.progarm-card {
+  background: #9b979751;
+  opacity: 0.6;
+}
+
+.progarm-card:hover {
+  border: 0.3rem solid #ee523c;
+}
+
+.bottom-para {
+  color: #fff;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  text-decoration: underline;
+  text-underline-offset: 5px;
+  padding: 4rem 0;
+  cursor: pointer;
+}
+
+/* featured speakers */
+
+.featured-speaker {
+  background-color: #f8f8f8;
+}
+
+.featured-image {
+  width: 100px;
+}
+
+.featured-heading {
+  color: #4f4f4f;
+  font-size: 2.4rem;
+  padding-top: 4rem;
+}
+
+.f-heading {
+  font-size: 2rem;
+  font-weight: 700;
+  color: #4f4f4f;
+}
+
+.featured-p1 {
+  color: #ee523c;
+  font-size: 1.5rem;
+}
+
+.featured-p2 {
+  color: #4f4f4f;
+  font-size: 1.5rem;
+}
+
+/* Partners */
+
+.Partners {
+  background-color: #272a31;
+}
+
+.partner-img {
+  width: 10rem;
+}
+
+/* footer section */
+
+.footer-logo {
+  width: 15rem;
+}
+
+.footer-para {
+  font-size: 1.3rem;
+  font-weight: 600;
+  color: #4f4f4f;
+}
+
+/* About page  */
+
+.main-heading-about {
+  color: #fa4524;
+  font-size: 5.5rem;
+  font-weight: 900;
+  line-height: 6rem;
+  text-transform: uppercase;
+}
+
+.info-about {
+  background-color: #f7f7f7;
+  border: #fff solid 0.5rem;
+  border-radius: 0.5rem;
+  font-size: 1.6rem;
+  color: #525252;
+  padding: 1rem 0;
+  margin: 5rem;
+}
+
+.main-about {
+  background-image: url("../images/bg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top left;
+  width: 100%;
+  height: 100vh;
+}
+
+.about-footer {
+  background-color: #272a31;
+}
+
+.footer-about {
+  color: #8a8888;
+}
+
+.past-para {
+  font-size: 1.4rem;
+  font-weight: 400;
+  color: #525252;
+}
+
+@media (min-width: 769px) {
+  .hamburger {
+    display: none;
+  }
+
+  .about-partner {
+    display: none;
+  }
+}
+
+@media (max-width: 769px) {
+  .top-links {
+    display: none;
+  }
+
+  .mylogo,
+  .nav-bttn {
+    display: none;
+  }
+
+  /* Hamburger */
+
+  .hamburger {
+    display: block;
+    position: absolute;
+    top: 2rem;
+    left: 3rem;
+    z-index: 2;
+  }
+
+  .hamburger.active .two {
+    opacity: 0;
+  }
+
+  .hamburger.active .one {
+    transform: translateY(10px) rotate(45deg);
+  }
+
+  .hamburger.active .three {
+    transform: translateY(-10px) rotate(-45deg);
+  }
+
+  .navbar_container {
+    position: fixed;
+    z-index: 2;
+    left: -150%;
+    top: 0;
+    gap: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    background-color: #e1e2f0;
+    background-color: #fff;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(17px);
+    width: 100%;
+    height: 100vh;
+    transition: 0.3s;
+    padding: 60px 20px;
+  }
+
+  .nav_links {
+    font-weight: 500;
+    font-size: 2.5rem;
+    line-height: 3.5rem;
+    color: #525252;
+    margin-bottom: 3rem;
+  }
+
+  .navbar_container.active {
+    left: 0;
+  }
```

Comment:
- [ ] **[Optional]** It would be better for the best user experience to add a background-color on the **navbar-container.active**  to avoid the text overlap :+1: :+1: 
example
```  
.navbar_container.active {
    left: 0;
    background-color: black;
  }
```

![image](https://user-images.githubusercontent.com/102757126/205455716-41559a02-19e8-43ae-816b-8adfba8feaeb.png)